osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmap.cs

```diff
@@ -288,6 +296,9 @@ public MenuItem[] ContextMenuItems
 
                 items.Add(new OsuMenuItem("Collections") { Items = collectionItems });
 
+                if (beatmapInfo.GetOnlineURL(api) is string url)
+                    items.Add(new OsuMenuItem("Copy link", MenuItemType.Standard, () => clipboard.SetText(url)));
```

Comment:
This doesn't have the OSD feedback like in `ExternalLinkButton`.

Reply:
May I ask about what the OSD stands for?

Reply:
You can google for an explanation. But please leave this alone, I'll fix it to keep things moving.

Reply:
`OnScreenDisplay`, but I dunno if it's better to copy the logic from `ExternalLinkButton` two more times, or add another method to `OsuGame`.

Reply:
Latter is more valid. I believe we will be using this in many more places going forward.